label_studio/data_manager/functions.py

```diff
@@ -6,13 +6,13 @@
 from django.conf import settings
 from rest_framework.generics import get_object_or_404
 
-from core.utils.common import int_from_request
-from data_manager.prepare_params import PrepareParams
-from data_manager.models import View
-from tasks.models import Task
+from label_studio.core.utils.common import int_from_request
+from label_studio.data_manager.prepare_params import PrepareParams
+from label_studio.data_manager.models import View
+from label_studio.tasks.models import Task
 
 
-TASKS = 'tasks:'
+TASKS = 'label_studio.tasks:'
```

Comment:
This should be unchanged, it's not relative to import

Reply:
hum, I've also renamed the `app_name` in `label_studio.tasks.urls` to `label_studio.tasks` ([here](https://github.com/heartexlabs/label-studio/pull/1344/files#diff-892c91f02e5dd7ba251171e84c88f4342900422c2fa42e27b76a1cc3caabae4dR8)) 

so if I'm not mistaken the goal is to use it to reverse find urls ? based on app_name ? or should I also revert the `app_name` in `label_sudio.tasks.urls` to just `tasks` ?